### DISCUSSION TITLE
Clicking a "New issue" link with pre-filled url params doesn't prefill those fields when the GitHub mobile app is installed

### DISCUSSION THREAD

**Discussion by derekantrican** [community/community#113726] [Mobile]
### Select Topic Area

Bug

### Body

I created a url for a new issue to prefill some fields by following the docs here: https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-url-query . However, if such a link is clicked on mobile and the GitHub app is also installed, it will prompt to select an issue template first (even if no template issues exist on the repository) and then none of the "prefilled fields" specified in the url will be carried over to the new issue form.

Example: on mobile (with the GitHub app installed) click this url: https://github.com/derekantrican/readdit/issues/new?title=New+bug+report&body=Describe+the+problem

Expected result: the GitHub app opens with the Title field prefilled with "New bug report" and the Description field prefilled with "Describe the problem"

Actual result: the GitHub app opens and prompts to select an issue template (even though none is present in the repository). After selecting "No template" the "Create Issue" window opens but all fields are blank

**Comment by esahmed21** [comment]
Thank you @derekantrican for reaching out and sharing your experience! Could you please confirm if you encountered this issue on the iOS or Android version of the mobile app?

**Reply by derekantrican** [reply]
Android. I'm on a Pixel 5 on the latest version of Android

**Reply by esahmed21** [reply]
Thank you for writing back! I'll create internal ticket for the Android team to look into it.

**Reply by naltatis** [reply]
Similar issue here. We've implemented a flow in our app/webUI that allows user the create a prefilled Github Issue. This works on the website but not when the iOS app is installed.
https://github.com/evcc-io/evcc/issues/23761

**Comment by TPS** [comment]
https://github.com/orgs/community/discussions/36533?
